saturn_client/core.py

```diff
@@ -0,0 +1,159 @@
+"""Python library for interacting with the Saturn Cloud API.
+
+NOTE: This is an experimental library and will likely change in
+the future
+"""
+
+import json
+import logging
+
+from typing import Any, Dict, Optional
+from urllib.parse import urljoin
+
+import requests
+from .settings import Settings
+
+
+log = logging.getLogger("saturn-client")
```

Comment:
```suggestion
log = logging.getLogger("saturn-client")
if log.level == logging.NOTSET:
    log.setLevel(logging.INFO)
```

(copied back from https://github.com/saturncloud/dask-saturn/pull/45)

Reply:
Thanks!